upload-override-jars

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+
+if [[ -n "$(git status --porcelain)" ]]; then
+	echo "You have uncommitted changes! Refusing to run."
+	exit 1
+fi
+
+branch_name="$(git rev-parse --abbrev-ref HEAD)"
+commit_hash="$(git rev-parse HEAD)"
+
+rsync -Pa jars/ foundry.remexre.xyz:/melt/jenkins/export-scratch/melt-jenkins/${commit_hash}-jars/
```

Comment:
Not super familiar with rsync, but will this work if the user from which the script is run doesn't happen to match the user on foundry?

Reply:
One can provide a username to use via `~/.ssh/config` (rsync just uses SSH under the hood)

Reply:
You could maybe add a comment to the script mentioning that.  I guess it's maybe not a huge issue anyway as only melt people should ever run this, and we can pick the account names on foundry.

Reply:
Added.